jwst/datamodels/model_base.py

```diff
@@ -7,9 +7,32 @@ class JwstDataModel(_DataModel):
 
     schema_url = "http://stsci.edu/schemas/jwst_datamodel/core.schema"
 
+    @property
+    def crds_observatory(self):
+        """
+        Get CRDS observatory code for this model.
+
+        Returns
+        -------
+        str
+        """
+        return "jwst"
+
+    def get_crds_parameters(self):
+        """
+        Get parameters used by CRDS to select references for this model.
+
+        Returns
+        -------
+        dict
+        """
+        return {
+            key: val for key, val in self.to_flat_dict(include_arrays=False).items()
+            if isinstance(val, (str, int, float, complex, bool))
```

Comment:
I assume this avoids grabbing an complex objects like `gwcs.WCS` objects from `meta` and even the data arrays?  Ideally, we would like something that works like `to_flat_dict()` but only on `self.meta`, right?

Reply:
Yes CRDS only accepts a handful of basic types (we'll be adding `astropy.time.Time` soon I think).